### PR TITLE
chore(deps-update): Bumped dependencies

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Various crucial and a non-crucial dependencies have been updated to mitigate existing vulnerabilities.
Following micro-services have been updated:

* `portal-api`.
* `tfm-api`.

## Resolution
* Updated `jswonwebtoken`.
* Updated `dicer`.
* Updated `apollo-server-express`.
* Updated `apollo-server-testing`.

### CODE REVIEW

**Review by Zainzzkk — APPROVED** [review]
LGTM

File: portal-api/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@azure/storage-file-share": "12.11.0",
     "@graphql-tools/schema": "^8.5.0",
-    "apollo-server-express": "2.18.2",
+    "apollo-server-express": "3.11.1",
```
Comment (IgnatG) [inline portal-api/package.json:38]: there are breaking changes in v3. we either update to the latest v2. or we need to fix the breaking changes

File: portal-api/package.json [hunk 2]
```diff
@@ -48,16 +48,16 @@
     "graphql": "^15.8.0",
     "graphql-middleware": "^6.1.33",
     "joi": "^17.6.0",
-    "jsonwebtoken": "8.5.1",
+    "jsonwebtoken": "^9.0.0",
```
Comment (IgnatG) [inline portal-api/package.json:51]: there are some breaking changes in v9 ->
https://github.com/auth0/node-jsonwebtoken/wiki/Migration-Notes:-v8-to-v9

File: portal-api/package.json (hunk 1 above)
Comment (abhi-markan) [reply portal-api/package.json]: Downgraded to 2.26.1

File: portal-api/package.json (hunk 2 above)
Comment (abhi-markan) [reply portal-api/package.json:51]: All tests have passed, are you happy for me to merge @IgnatG ?

Comment (IgnatG) [reply portal-api/package.json:51]: yup - happy to merge

**Review by IgnatG — APPROVED** [review]
LGTM

**Pull request merged**